packages/commons/src/AxePartialResult.cs

```diff
@@ -0,0 +1,139 @@
+using System.Collections.Generic;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+
+namespace Deque.AxeCore.Commons
+{
+    /// <summary>
+    /// Results returned from axe.runPartial
+    /// </summary>
+    [JsonObject(NamingStrategyType = typeof(CamelCaseNamingStrategy))]
+    public class AxePartialResult
```

Comment:
Similar to the comment in `AxeRunContext.cs`: the axe-core API docs and the axe-core `doc/run-partial.md` leave the axe partial result type opaque and only document that it's suitable for passing as-is to `axe.finishRun`. I think it would be preferable for the opaque-ness to be consistent between this library and the axe-core docs, ie, don't bother deserializing/reserializing the partial results to strong object types at all and instead just pass them as-is back to `runPartial` (as a list of either `JObject` or `JRaw`)